.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+offerings:
+  - name: deploy-arch-ibm-cloud-logs
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: 63d8ae58-fbf3-41ce-b844-0fb5b85882ab
+    variations:
+      - name: fully-configurable
+        mark_ready: true
+        install_type: fullstack
+        # scc:
+          # instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
+          # region: us-south
+          # scope_resource_group_var_name: existing_resource_group_name
+      - name: security-enforced
+        mark_ready: true
+        install_type: fullstack
+        # scc:
+          # instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
+          # region: us-south
+          # scope_resource_group_var_name: existing_resource_group_name
```

Comment:
The ibm_catalog.json should have a compliance section in it which claims from the FSCloud profile just like we do for all other DAs, and the above scc section should be uncommented